plugins/core/tracer_ignore.go

```diff
@@ -0,0 +1,133 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package core
+
+import (
+	"strings"
+)
+
+func tracerIgnore(operationName string, ignoreSuffixList, ignorePath []string) bool {
+	return ignoreSuffix(operationName, ignoreSuffixList) || traceIgnorePath(operationName, ignorePath)
+}
+
+func ignoreSuffix(operationName string, ignoreSuffix []string) bool {
+	suffixIdx := strings.LastIndex(operationName, ".")
+	if suffixIdx == -1 {
+		return false
+	}
+	for _, suffix := range ignoreSuffix {
+		if suffix == operationName[suffixIdx:] {
+			return true
+		}
+	}
+	return false
+}
+
+func traceIgnorePath(operationName string, ignorePath []string) bool {
+	for _, pattern := range ignorePath {
+		if normalMatch(pattern, 0, operationName, 0) {
+			return true
+		}
+	}
+	return false
+}
+
+func normalMatch(pat string, p int, str string, s int) bool {
```

Comment:
Can you add more comments to the parameters? This can help developers better understand your code.

Reply:
Done.